ExampleUsage/EnumExample.cs

```diff
@@ -99,6 +141,14 @@ public enum Size
 	public enum Shape
 	{
 		Square,
-		Round
+        [System.ComponentModel.Description("Round it is!")]
+        Round
 	}
+
+    public class Enum
```

Comment:
hmm, so you would. I think leave it as you have it then, especially as this is the example usage and it's better that that's self-contained. I think the name Enum is liable to confuse people so I'd think of a different name. I'm having a play with the code now.